src/e2e-test/java/io/cdap/plugin/common/stepsdesign/TestSetupHooks.java

```diff
@@ -924,4 +924,111 @@ public static void createSourceBQUpdateTable() throws IOException, InterruptedEx
   public static void createGcsBucketWithAvro() throws IOException, URISyntaxException {
     gcsSourceBucketName = createGCSBucketWithFile(PluginPropertyUtils.pluginProp("gcsAvroAllDataFile"));
   }
+
+  @Before(order = 1, value = "@BQ_UPSERT_SOURCE_TEST")
```

Comment:
Add the after hooks for each of these before hooks, to cleanup the created tables after the scenario execution.

Reply:
resolved